plugins/modules/gitlab_group_variable.py

```diff
@@ -99,6 +99,12 @@
           - The scope for the variable.
         type: str
         default: '*'
+      raw:
+        description:
+          - Whether the variable is expandable.
```

Comment:
This is a new feature, not really a bugfix. It should also be able to fix this without adding a new feature (by simply ignoring unknown properties when doing an idempotence check). (Just a quick comment, haven't had time yet to read the remainder of the discussion :) )

Reply:
Yeah, that's a good point. I try to rework it.